source/templates.tex

```diff
@@ -2161,7 +2162,12 @@
 A template declaration in which the name of the template is followed by
 a \grammarterm{template-argument-list} is
 a partial specialization of
-the template named in the declaration\iref{temp.class.spec}.
+the template named in the declaration\iref{temp.spec.partial}.
+\begin{note}
+A \grammarterm{template-id}
+cannot be the declared name of a primary template declaration.
+However, this syntax is allowed in partial specializations\iref{temp.spec.partial}.
+\end{note}
```

Comment:
This note feels very much redundant with the normative text in front of it. Please drop.

Reply:
Done, thanks!